src/registrar/tests/common.py

```diff
@@ -88,6 +88,37 @@ def send_email(self, *args, **kwargs):
         self.EMAILS_SENT.append({"args": args, "kwargs": kwargs})
 
 
+def mock_user():
+    """A simple user."""
+    user_kwargs = dict(
+        id=4,
+        first_name="Rachid",
+        last_name="Mrad",
+    )
+    mock_user, _ = User.objects.get_or_create(**user_kwargs)
+    return mock_user
+
+
+def create_superuser(self):
```

Comment:
self is never used here or below in create_user, I think it can be removed unless there is some other reason it was added. Also normally if you have a utility function, I wouldn't pass "self" because "self" is like "this" in js and usually is meant to convey a special meaning (ie pass the class this function resides in). Since this isn't a class, I would have my argument describe what it's for.